src/propcache/_helpers_py.py

```diff
@@ -46,13 +46,13 @@ def __init__(self, wrapped: Callable[..., _T]) -> None:
         self.name = wrapped.__name__
 
     @overload
-    def __get__(self, inst: None, owner: Optional[Type[Any]] = None) -> Self: ...
+    def __get__(self, inst: None, owner: Optional[Type[object]] = None) -> Self: ...
 
     @overload
-    def __get__(self, inst: _TSelf[_T], owner: Optional[Type[Any]] = None) -> _T: ...
+    def __get__(self, inst: _TSelf[_T], owner: Optional[Type[object]] = None) -> _T: ...
```

Comment:
It looks like the union of these ended up being what's declared below already. Perhaps, they just need to be deleted?